resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/internal/properties/PropertyManagerTest.java

```diff
@@ -550,4 +555,42 @@ public void testSimpleUpdate() throws CoreException {
 		}
 	}
 
+	/**
+	 * Whenever a delete operation is called on an IFile it's properties also
+	 * deleted from .index file .This Test case validates for given IFile resource
+	 * Zero depth is calculated to traverse through folders for loading right index
+	 * file and delete its properties . because the required index file is present
+	 * under corresponding bucket of the folder same as the IFile and no need to
+	 * traverse to Infinite depth.
+	 */
+
+	@Captor
+	ArgumentCaptor<IResource> resourceArgCaptor;
+	@Captor
+	ArgumentCaptor<Integer> depthArgCapture;
+
+	@Test
+	public void test() throws CoreException {
```

Comment:
Please give the test case a more meaningful name, such as `testFileDeleteTraversalDepth`.